### PR TITLE
Release 5.0.0 - Externally scheduled sync

### PR DESCRIPTION
**Pull request by briskt**
### Changed
- Changed the container to run `/data/yii batch/full` instead of the cron service. This requires a change to the deployment of this service. Rather than using a constantly-running ECS service, it should be triggered to run on a schedule by Cloudwatch or something similar.

### Removed
- Removed frontend and Apache server
- Removed `.scrutinizer.yml`
- Removed `s3-expand`
- Removed `Vagrantfile`

---

### Release PR Checklist
- [x] Update version number in main.php Sentry configuration

### CODE REVIEW

**Comment by briskt** [comment]
> Soooo long. I do recall seeing these changes prior to release, and a couple I did approve, so I'll approve this.

Thank you. Since this is interdependent with https://github.com/silinternational/idp-in-a-box/pull/234 I'll hold off and wait until that one is approved.

**Review by forevermatt — APPROVED** [review]

**Pull request merged**
